Plugins/Source/KISAddonPointer.cs

```diff
@@ -12,8 +12,8 @@ namespace KIS {
 
 [KSPAddon(KSPAddon.Startup.Flight, false)]
 sealed class KISAddonPointer : MonoBehaviour {
-  public GameObject audioGo = new GameObject();
-  public AudioSource audioBipWrong = new AudioSource();
+  public GameObject audioGo = null;
+  public AudioSource audioBipWrong = null; //new AudioSource();
```

Comment:
Comments get lost in the code very quickly since it's unusual for people to review own code once it's submitted :) If mistake is not caught during the review then it turns into bug.